### ISSUE TITLE
Add outlines to focused buttons for accessibility

### ISSUE THREAD

**Issue by llf5185** [RIT-Software-Engineering/RIT-SE-Senior-Project#325]
Due to Semantic UI, the typical behavior of outlining a button in focus is replaced with changing the saturation and lightness of it. This may be unclear in cases where there is only one button of a particular color on the page or, as stated in the following WCAG 2.2 Understanding document https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance, if the user is using magnification tools. In both cases, it becomes difficult to compare a focused element with one in an unfocused state. Additionally, Semantic UI's approach of using saturation and lightness to indicate focus completely fails when the button is transparent and thus lacking color, such as with the buttons on the navigation bar in the current dev version, thus providing no focus indicator.

These difficulties significantly affect navigating the website on a keyboard, since they make it harder to keep track of what you are currently selecting. This issue proposes that all buttons on the SE Senior Project website have their outline readded when focused.
